apps/src/StudioApp.js

```diff
@@ -802,6 +802,11 @@ StudioApp.prototype.showInstructions_ = function(level, autoClose) {
   }
 
   dialog.show({hideOptions: hideOptions});
+
+  if (renderedMarkdown && $.fn.details) {
```

Comment:
Good point. My initial thinking was just in case this code would be triggered in some context (tests or test page) without the library available, just because it's being included via dashboard's jQuery which felt a bit loose. Since much of the code in the show instructions flow assumes jQuery, though, I'll remove it.